fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -191,9 +191,9 @@ ACTOR Future<Void> getRate(UID myID, Reference<AsyncVar<ServerDBInfo>> db, int64
 			}
 			reply = brokenPromiseToNever(db->get().ratekeeper.get().getRateInfo.getReply(
 			    GetRateInfoRequest(myID, *inTransactionCount, *inBatchTransactionCount, tagCounts,
-			                       *transactionTagCommitCostEst, detailed)));
+			                       *ssTagCommitCost, detailed)));
 			transactionTagCounter->clear();
-			transactionTagCommitCostEst->clear();
+			ssTagCommitCost->clear();
```

Comment:
Was there some comment thread I missed about why the change to `ssTag` in the name here?  I'm still just incapable of reading "storage server tag" and not thinking of ... tags for storage servers.

Reply:
Because proxy keeps `unordered_map<TransactionTag, Cost>` for each storage server. Would `ssTrTagCommitCost` help a little bit?